home/header.asm

```diff
@@ -170,4 +170,6 @@ if DEF(ANALOGUE_POCKET)
 endc
 
 	; The rest of the header is handled by rgbfix.
-	ds $0150 - @, $00
+	ds $014e - @, $00
+
+RomChecksum:: ds 2
```

Comment:
```suggestion
RomHeaderChecksum::
	ds 2, $00
```

Reply:
There's a separate checksum for the header alone at $14d, while $14e-$14f covers all banks